release-1.9/release-1.9.md

```diff
@@ -0,0 +1,100 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
+
+The proposal below is similar in layout previous releases:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- there are a number of North American holidays during Q4, so this release will
+  have reduced scope and fewer features
+- same as in 1.8
+  * alpha's should not be cut unless all master-release-blocking tests are passing
+  * release notes for planned features done 2 weeks out from feature freeze
+  * docs PRs deadlines ahead of release
+
+## 1.9 Release Schedule Key Dates
+- Oct 5nd (Weds) coding start
+- Oct 11th (Weds): v1.9.0-alpha.1
+- Oct 25th (Weds): v1.9.0-alpha.2
+- Oct 27th (Fri): Feature Freeze
+- Nov 8th (Weds): v1.9.0-alpha.3
+- Nov 22nd (Weds): v1.9.0-alpha.4
+- Nov 22nd (Weds): Code Freeze 
+- Nov 29th (Weds): v1.9.0-beta.0, v1.10.0-alpha.0
```

Comment:
note that the `release-1.9` branch is set up and people should start adding v1.9-targeted tests.